sdk/storage/azblob/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/Azure/azure-sdk-for-go/sdk/storage/azblob
+
+go 1.17
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.19.0
+	github.com/Azure/azure-sdk-for-go/sdk/internal v0.7.0
+	github.com/Azure/azure-sdk-for-go/sdk/to v0.1.4
```

Comment:
The `to` module has been deprecated, can you switch to using `azcore/to` instead.